components/coachmark/index.css

```diff
@@ -10,127 +10,157 @@ OF ANY KIND, either express or implied. See the License for the specific languag
+	--spectrum-coachmark-title-text-font-weight: var(--spectrum-heading-sans-serif-font-weight);
+	--spectrum-coachmark-title-font-size: var(--spectrum-coach-mark-title-size);
+	--spectrum-coachmark-title-text-line-height: var(--spectrum-heading-line-height);
+
+	--spectrum-coachmark-content-font-color: var(--spectrum-body-color);
+	--spectrum-coachmark-content-font-weight: var(--spectrum-body-sans-serif-font-weight);
+	--spectrum-coachmark-content-font-family: var(--spectrum-body-sans-serif-font-family);
+	--spectrum-coachmark-content-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-coachmark-content-line-height: var(--spectrum-body-line-height);
+	--spectrum-coachmark-content-font-size: var(--spectrum-coach-mark-body-size);
+
+	--spectrum-coachmark-step-color: var(--spectrum-coach-mark-pagination-color);
+	--spectrum-coachmark-step-font-weight: var(--spectrum-body-medium-font-weight);
+	--spectrum-coachmark-step-font-family: var(--spectrum-body-sans-serif-font-family);
+	--spectrum-coachmark-step-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-coachmark-step-line-height: var(--spectrum-body-line-height);
+	--spectrum-coachmark-step-font-size: var(--spectrum-coach-mark-pagination-body-size);
+	--spectrum-coachmark-step-to-bottom: var(--spectrum-coach-mark-pagination-text-to-bottom-edge);
+
+	/* mods */
+	--mod-buttongroup-justify-content: flex-end;
+	--mod-popover-border-width: var(--spectrum-border-width-100);
+	--mod-popover-corner-radius: var(--spectrum-corner-radius-100);
+	--mod-popover-content-area-spacing-vertical: 0;
+	--mod-button-edge-to-visual-only: 9px;
+}
 
-.spectrum-CoachMarkPopover {
+.spectrum-CoachMark {
   position: relative;
-
-  min-inline-size: var(--spectrum-coachmark-min-width);
-  max-inline-size: var(--spectrum-coachmark-max-width);
-
-  border-radius: var(--spectrum-coachmark-border-radius);
-  border-width: var(--spectrum-coachmark-border-size);
-  border-style: solid;
-
+	min-inline-size: var(--mod-coachmark-min-width, var(--spectrum-coachmark-min-width));
+	max-inline-size: var(--mod-coachmark-max-width, var(--spectrum-coachmark-max-width));
+	inline-size: var(--mod-coachmark-width, var(--spectrum-coachmark-width));
 }
-.spectrum-CoachMarkPopover-image {
-  border-radius: var(--spectrum-coachmark-border-radius) var(--spectrum-coachmark-border-radius) 0 0;
 
-  inline-size: 100%;
-}
+.spectrum-CoachMark-buttongroup {
+	display: var(--spectrum-coachmark-buttongroup-display);
 
-.spectrum-CoachMarkPopover-header,
-.spectrum-CoachMarkPopover-content,
-.spectrum-CoachMarkPopover-footer {
-  padding-block: 0;
-  padding-inline: var(--spectrum-coachmark-padding-x);
+	&--mobile {
+ 		--mod-buttongroup-spacing-horizontal: var(--spectrum-spacing-100);
+		display: var(--spectrum-coachmark-buttongroup-mobile-display);
+	}
 }
 
-.spectrum-CoachMarkPopover-header {
-  padding-block-start: var(--spectrum-coachmark-padding-y);
-}
+.spectrum-CoachMark-menu {
```

Comment:
Can you tell me where are you using this class?

Reply:
here: https://github.com/adobe/spectrum-css/pull/1992/files#diff-9de3cfd8cfc3c8ef2762992562607c221755d62782a669cec2afd19420d12c1fR54 

Using the same pattern as the button group here but for the different sizes action menus.